### DISCUSSION TITLE
recharge by hydrological year

### DISCUSSION THREAD

**Discussion by VeloVolant** [pastas/pastas#659] [Q&A]
Hello ! I am using this little code to print annual recharge resulting from Pastas simulations.
print (ml.get_stress('recharge').resample("Y").sum())

Could you suggest a similar code to get the "hydrological year" recharge, e.g. from 1rst of august to 31st of jully ?
Regards, Didier

**Comment by mbakker7** [comment]
Dear Didier,
Thanks for your question.
This is a pandas question, as `resample` is a pandas function. Please post the question on the `pandas` website (or on stack overflow). Or check the `resample` docs from pandas.

**Comment by VeloVolant** [comment]
Thank your very much Mark ! Regards
